### DISCUSSION TITLE
About scene description using USDz or GLTF2

### DISCUSSION THREAD

**Discussion by fanoux** [mitsuba-renderer/mitsuba3#762] [Feature requests] [answered]
Hi,

Mitsuba is really awesome renderer.
I am wondering if there are plans to support ascii based descriptions like USDA with MDL to load a scene ?
Not sure about how PBR and especially normal map info could be used but as a first step be able to get the geometry with also MDL would be convenient.
Because with this it would allow to use most of the mainstream modeling tools to craft a scene.

**Comment by wjakob** [comment, marked as answer]
It's beyond the scope of this project, which is focused on research and not production rendering.
